lib/ocpp/v16/charge_point_configuration.cpp

```diff
@@ -219,47 +219,47 @@ std::string ChargePointConfiguration::getChargeBoxSerialNumber() {
 }
 
 CiString<20> ChargePointConfiguration::getChargePointModel() {
-    return CiString<20>(this->config["Internal"]["ChargePointModel"]);
+    return CiString<20>(this->config["Internal"]["ChargePointModel"].get<std::string>());
```

Comment:
Can't you use the new constructor for this now? I'm not sure if both changes are needed.

Reply:
I've removed the json constructor, so these changes make a lot more sense now